UsabillaSampleApp/README.md

```diff
@@ -0,0 +1,29 @@
+[![license](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://github.com/usabilla/usabilla-u4a-react-native/blob/develop/LICENSE)
+
+# Usabilla for Cordova
+
+## Resource Generator
+
+This tool will crop and resize JPEG and PNG source images to generate icons and splash screens for modern iOS, Android, and Windows. `cordova-res` was developed for use with Cordova, but Capacitor and other native runtimes are supported.
```

Comment:
modern iOS, Android and Windows... I guess `phones`?

then:
 was developed for use with Cordova ->  was developed to be used with Cordova

Reply:
This is a copied text from the actual cordova-res repository.

Reply:
So we do not want to change it?

Reply:
Ya, we can as its in our repository, but than it will be changed as its mentioned in there official repository. so i just thought does it matter.

Reply:
Done